demisto_sdk/commands/common/hook_validations/dashboard.py

```diff
@@ -136,11 +136,10 @@ def is_including_fields(self) -> bool:
                             error_message,
                             error_code,
                             file_path=self.file_path,
-                            should_print=False,
                         )
                         if formatted_message:
                             is_valid = False
                             error_msg += formatted_message
         if error_msg:
-            print_error(error_msg)
+            logger.info(f"[red]{error_msg}[/red]")
```

Comment:
shouldn't we use `logger.error`?

Reply:
print_error was just adding colors.
I'll change it to logger.error, to be consistent.